src/pages/ProductDetail/ProductDetail.js

```diff
@@ -1,7 +1,285 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import * as thousand from '../../styles/thousand';
+import theme from '../../styles/theme';
+import {
+  faHeart,
+  faCartShopping,
+  faGift,
+  faBookOpen,
+} from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import PaymentModal from './PaymentModal';
+import StarRating from './StarRating';
 
 const ProductDetail = () => {
-  return <h1>hi</h1>;
+  const [book, setBook] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/detail.json')
+      .then(res => res.json())
+      .then(data => setBook(data));
+  }, []);
+
+  const [isOpen, setOpen] = useState(false);
```

Comment:
state는 한 곳에서 관리해주세요!
```suggestion
  const [book, setBook] = useState([]);
  const [isOpen, setOpen] = useState(false);

  useEffect(() => {
    fetch('/data/detail.json')
      .then(res => res.json())
      .then(data => setBook(data));
  }, []);
```